pokedex.html

```diff
@@ -0,0 +1,16 @@
+<html>
```

Comment:
File should be named index.html if it is the landing page.  Github pages only accepts this naming convention when deploying sites.